examples/synopsysctl/securityContext.json

```diff
@@ -4,6 +4,15 @@
         "runAsUser": 1000,
         "runAsGroup": 1000
     },
+    "blackduck-postgres": {
+        "fsGroup": 26,
+        "runAsUser": 26,
+        "runAsGroup": 26
+    },
+    "blackduck-init": {
```

Comment:
Still we need to have different security context for blackduck-postgres and blackduck-init?